### PR TITLE
Layout harmonization

### PR DESCRIPTION
**Pull request by pgiraud**
Some more work on the harmonization.
This time on the tools panels (print & draw).

screenshot from 2016-06-09 11 47 09
screenshot from 2016-06-09 11 47 49
screenshot from 2016-06-09 11 48 06

### CODE REVIEW

File: contribs/gmf/less/vars.less [hunk 1]
```diff
@@ -30,3 +30,4 @@
 
 @brand-primary: #b1c2b5;
 @brand-secondary: #d3e5d7;
+@font-size-small: ceil(@font-size-base * 0.9);
```
Comment (sbrunner) [inline contribs/gmf/less/vars.less]: I think that we shouldn't use ceil and floor to be ready for bootstrap 4...
https://github.com/twbs/bootstrap/pull/17021

**Comment by sbrunner** [comment]
Just one comment, also that's looks good :-)

File: contribs/gmf/less/vars.less (hunk 1 above)
Comment (pgiraud) [inline contribs/gmf/less/vars.less]: Can you please elaborate what the link between ceil/floor and bootstrap 4 is?

Comment (sbrunner) [inline contribs/gmf/less/vars.less]: Bootstrap 4 will use rem, than a ceil or a floor will do bad things (e.g. round on 10 pixels...)

See « Dropped IE8 support and moved to rem and em units. » on http://blog.getbootstrap.com/2015/08/19/bootstrap-4-alpha/

Comment (pgiraud) [inline contribs/gmf/less/vars.less]: Makes sense. Thanks. Changed.

**Comment by pgiraud** [comment]
Final review appreciated.
Demo: https://pgiraud.github.io/ngeo/layout_harmonization/examples/contribs/gmf/apps/desktop

**Comment by fredj** [comment]
LGTM, thanks

**Pull request merged**